### ISSUE TITLE
[Manure] Perform a mass balance assessment of the Manure Module 

### ISSUE THREAD

**Issue by elle-andreen** [RuminantFarmSystems/RuFaS#1802]
**Is your feature request related to a problem? Please describe.**
A key principal of biological and physical processes is conservation of mass. In the Manure Module, manure undergoes many physical and chemical changes, including degradation and transformation of substrates (e.g. degrading organic matter into acids, CO2, and CH4), as well as more macro-level processes like solid-liquid separation. At both scales, it is essential that total and individual nutrient or component masses are preserved or accounted for throughout the manure management chain. 

**Describe the solution you'd like**
We should assess this in RuFaS by conducting a mass balance analysis. This entails quantifying all of our tracked nutrient masses as-excreted from the Animal Module, and tracking their respective gains, losses, and transformations through the manure management chain. If we are conserving mass correctly, the total input quantity (e.g., amount excreted by animals plus any amount gained from environment/management, such as bedding addition to manure) should be equal to the end-point quantity, minus quantified losses or transformation. I believe we can call field application or export the end-point for manure module purposes.

Where we have mass balance inconsistencies, these should be noted and corrected. 

**Additional context**
As we're working through this process, we should also be considering issue #1645, which is related to where and how we update manure nutrient gains and losses throughout the manure management chain. 

Mass balance is an important part of module evaluation - we should perform this in a standardized way that can be easily shared and understood. Additionally, if major changes are made to the module (e.g., PR #1623 and related PRs), this process should be repeated. We should assess mass balance as efficiently as possible while configuring the process to be re-usable for future evaluations.

**Comment by elle-andreen** [comment]
Adding for future reference:

In terms of liquid manure total N, the calculation in SSO/U and anaerobic lagoon currently is:
N2O kg/d = (daily Manure N - daily ammonia loss) * N2O factor 

I believe the above is incorrect; the N2O factor was developed to be applied to the daily manure N entering storage, not including storage ammonia losses ([see Kristan's original instructions here](https://docs.google.com/document/d/1ny5C6ZveCocXKytIvceNsCsp3HNTyJSTjZpoTqNUodw/edit)). Currently the ammonia loss is being subtracted from manure N value that is then used to calculate N2O. This ammonia value is representative of **the entire manure storage**, not that single days' worth of manure. 

Current functionality:
Recep pit N = **87 kg**
Manure treatment ammonia loss = 23 kg
Manure treatment N2O = **(87 kg - 23 kg)** * 0.005 (for covered outdoor slurry) = 0.32
Manure treatment N = (87 kg - 23 kg - 0.32) = 63.7

Correct N2O(?) = **87 kg** * 0.005 = 0.435

**Comment by elle-andreen** [comment]
SME grooming 2/27/26: The core request of this issue (perform a mass balance assessment to confirm conservation of mass through the manure chain) is still valid. Some problems and references mentioned in the issue are outdated. This issue should be tackled by a SME, and any resulting violations of conservation of mass should be either documented, or if fixable, have issues created for the dev team to address.
